group_vars/mons.yml.sample

```diff
@@ -45,9 +45,20 @@ dummy:
 #cephfs_metadata: cephfs_metadata # name of the metadata pool for a given filesystem
 
 #cephfs_pools:
-#  - { name: "{{ cephfs_data }}", pgs: "" }
-#  - { name: "{{ cephfs_metadata }}", pgs: "" }
-
+#  - name: "{{ cephfs_data }}"
```

Comment:
you need to edit `roles/ceph-mon/defaults/main.yml` then run `generate_group_vars_sample.sh`

Reply:
Good point :/

Fixed, thanks.